src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -529,6 +535,29 @@ export default {
 				'b',
 			);
 		}
+
+		// CASH-521: Hover loan card experiment
```

Comment:
Could you please move all of this setup into it's own method similar to how initializeFavoriteCountryRowExp is handled?

Reply:
+1

Reply:
@mcstover @emuvente Cool, encapsulated the code :+1: